problem_builder/tests/integration/test_mentoring.py

```diff
@@ -165,18 +172,23 @@ def _standard_checks(self, answer, mcq, mrq, rating, messages):
         )
         self._assert_feedback_showed(mrq, 2, "This MRQ is indeed very graceful", click_choice_result=True)
         self._assert_feedback_showed(mrq, 3, "Nah, there aren't any!", click_choice_result=True, success=False)
+        self.scroll_to(rating)
         self._assert_feedback_showed(rating, 3, "I love good grades.", click_choice_result=True)
         self.assertTrue(messages.is_displayed())
+        self.scroll_to(messages)
         self.assertEqual(messages.text, "FEEDBACK\nNot done yet")
 
     def _feedback_customized_checks(self, answer, mcq, mrq, rating, messages):
         # Long answer: Previous answer and feedback visible
+        self.scroll_to(answer)
```

Comment:
@jbzdak Selenium *should* be doing this scrolling - is it possible that the elements in question were just hidden (in the process of becoming visible), and this scroll_to has a side effect of fixing it since it acts as a short delay?

These `scroll_to` calls seem very hacky, and I'd rather find a more direct way of addressing the problem. 

Please try this way of fixing it instead (which is a nice cleanup, if it works): 351705cb5caa8425858b013d958fba1b8b84b3c2